app/errors.py

```diff
@@ -0,0 +1,22 @@
+class NotWearingMaskError(Exception):
+    """
+    Not wearing mask
+    """
+
+
+class VaccineError(Exception):
```

Comment:
Consider providing a default error message for your exception. For example: `class VaccineError(Exception): def __init__(self, message='There is a problem with the vaccine'): super().__init__(message)`